### PR TITLE
Session lazy psr16

### PR DESCRIPTION
**Pull request by prisis**
| Question      | Answer
| ------------- | ------
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| License       | MIT

### Description
added the missing psr-16 SessionHandler
implemented PHP 7.0's `SessionUpdateTimestampHandlerInterface` with a new `AbstractSessionHandler` base class

### TODO
* [x] Add tests
* [ ] Add documentation
* [x] Updated Changelog.md

### CODE REVIEW

**Comment by prisis** [comment]
Okay travis is broken for me nice ...

File: src/SessionHandler/Psr6SessionHandler.php [hunk 1]
```diff
@@ -36,41 +37,63 @@ class Psr6SessionHandler implements \SessionHandlerInterface
...
+    public function close()
     {
         return true;
     }
 
     /**
      * {@inheritdoc}
      */
-    public function close()
+    public function gc($lifetime)
```
Comment (Nyholm) [inline src/SessionHandler/Psr6SessionHandler.php]: Shouldnt this part of the AbstractSessionHandler?

File: src/SessionHandler/Psr6SessionHandler.php [hunk 2]
```diff
@@ -36,41 +37,63 @@ class Psr6SessionHandler implements \SessionHandlerInterface
...
+
         $this->ttl    = isset($options['ttl']) ? (int) $options['ttl'] : 86400;
         $this->prefix = isset($options['prefix']) ? $options['prefix'] : 'psr6ses_';
     }
 
     /**
      * {@inheritdoc}
      */
-    public function open($savePath, $sessionName)
+    public function close()
```
Comment (Nyholm) [inline src/SessionHandler/Psr6SessionHandler.php]: Shouldnt this part of the AbstractSessionHandler?

File: src/SessionHandler/Psr6SessionHandler.php (hunk 1 above)
Comment (prisis) [reply src/SessionHandler/Psr6SessionHandler.php]: i just forgot it

**Pull request merged**